integration-tests/jpa-postgresql/src/main/java/io/quarkus/it/vault/Address.java

```diff
@@ -1,4 +1,4 @@
-package io.quarkus.it.jpa.postgresql;
+package io.quarkus.it.vault;
```

Comment:
Is it Git doing weird things or did you remove the files from the `jpa-postgresql` module?

Reply:
bad (and wild) package refactoring with intellij. thanks for picking that up. I have reverted it.